src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediaentryeditor/mediaentryeditor.html

```diff
@@ -0,0 +1,118 @@
+<div class="umb-media-entry-editor"  ng-controller="Umbraco.Editors.MediaEntryEditorController as vm">
+
+    <ng-form name="vm.imageCropperForm" val-form-manager>
+
+        <umb-editor-view umb-tabs ng-if="!page.loading">
+
+            <umb-editor-header
+                name="vm.media.name"
+                name-required="false"
+                name-locked="true"
+                hide-alias="true"
+                hide-icon="true"
+                hide-description="true">
+            </umb-editor-header>
+
+            <div class="umb-editor-container umb-panel-body umb-scrollable">
```

Comment:
It can use `<umb-editor-container>` here.
I recently fixed this in a few PRs:
Block List editor: https://github.com/umbraco/Umbraco-CMS/pull/9664
Block List overlay: https://github.com/umbraco/Umbraco-CMS/pull/9831